src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,13 +1,53 @@
 package core.mate.academy.model;
 
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
 /**
  * Add some custom fields that could be only in Bulldozer
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String model;
+    private Date year;
+    private double price;
+
     public Bulldozer() {
     }
 
+    public List<Bulldozer> get() {
+        List<Bulldozer> bulldozers = new ArrayList<>();
+        bulldozers.add(new Bulldozer());
+        bulldozers.add(new Bulldozer());
+        bulldozers.add(new Bulldozer());
+        return bulldozers;
+    }
```

Comment:
This method should not be here. It should be in BulldozerProducer.
Also fix it in Excavator and Truck classes
```suggestion
```